Failed to skip private modules during publishing

<!--- Provide a general summary of the issue in the Title above -->

#### Summary / Current Behavior

The publish script is throwing ERR instead of bypassing private modules as expected.

```
npm ERR! This package has been marked as private
npm ERR! Remove the 'private' field from the package.json to publish it.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/gnavvy/.npm/_logs/2020-01-10T19_31_04_771Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! manifold@1.1.3 lerna:publish: `lerna exec cp package.json dist && lerna exec npm publish dist`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the manifold@1.1.3 lerna:publish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/gnavvy/.npm/_logs/2020-01-10T19_31_04_804Z-debug.log
error Command failed with exit code 1.
```

#### Expected Behavior

Modules marked as private should be skipped during publishing.

#### Context and Environment

We currently have a few and will keep adding experimental modules into this repo. It is desired to exclude them from being published to NPM in early development. One viable solution is to use the `"private": true` flag in the module's package.json (example) to bypass the publish step, but this breaks `npm publish`.

#### Steps to Reproduce

To reproduce, run the script in the following order
```
npm run build
npm run publish
```
#### Possible Solution

Explicitly adding the `--no-private` flag after `lerna exec` during publish bypasses `npm publish` inside the private modules.

--- a/bindings/jupyter-modules/jupyter-graph-builder/package.json
+++ b/bindings/jupyter-modules/jupyter-graph-builder/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-graph-builder",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/graph-builder": "^1.1.4-alpha.5",
+    "@mlvis/graph-builder": "^1.1.4",
     "react": "^16.9.0"
   }
 }

--- a/bindings/jupyter-modules/jupyter-graph-builder/package.json
+++ b/bindings/jupyter-modules/jupyter-graph-builder/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-graph-builder",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/graph-builder": "^1.1.4-alpha.3",
+    "@mlvis/graph-builder": "^1.1.4-alpha.4",
     "react": "^16.9.0"
   }
 }

--- a/bindings/jupyter-modules/jupyter-graph-builder/package.json
+++ b/bindings/jupyter-modules/jupyter-graph-builder/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-graph-builder",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/graph-builder": "^1.1.3",
+    "@mlvis/graph-builder": "^1.1.4-alpha.0",
     "react": "^16.9.0"
   }
 }

--- a/bindings/jupyter-modules/jupyter-graph-builder/package.json
+++ b/bindings/jupyter-modules/jupyter-graph-builder/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-graph-builder",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/graph-builder": "^1.1.4-alpha.2",
+    "@mlvis/graph-builder": "^1.1.4-alpha.3",
     "react": "^16.9.0"
   }
 }

--- a/bindings/jupyter-modules/jupyter-graph-builder/package.json
+++ b/bindings/jupyter-modules/jupyter-graph-builder/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-graph-builder",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/graph-builder": "^1.1.4-alpha.1",
+    "@mlvis/graph-builder": "^1.1.4-alpha.2",
     "react": "^16.9.0"
   }
 }

--- a/bindings/jupyter-modules/jupyter-graph-builder/package.json
+++ b/bindings/jupyter-modules/jupyter-graph-builder/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-graph-builder",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/graph-builder": "^1.1.4-alpha.4",
+    "@mlvis/graph-builder": "^1.1.4-alpha.5",
     "react": "^16.9.0"
   }
 }

--- a/bindings/jupyter-modules/jupyter-graph-builder/package.json
+++ b/bindings/jupyter-modules/jupyter-graph-builder/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-graph-builder",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/graph-builder": "^1.1.4-alpha.0",
+    "@mlvis/graph-builder": "^1.1.4-alpha.1",
     "react": "^16.9.0"
   }
 }

--- a/bindings/jupyter-modules/jupyter-ma-causal/package.json
+++ b/bindings/jupyter-modules/jupyter-ma-causal/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-ma-causal",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",

--- a/bindings/jupyter-modules/jupyter-ma-causal/package.json
+++ b/bindings/jupyter-modules/jupyter-ma-causal/package.json
@@ -7,8 +7,6 @@
     "url": "git+https://github.com/uber/manifold.git"
   },
   "main": "index.js",
-  "-------- Avoid publishing the prototype for now --------": "",
-  "private": true,
   "publishConfig": {
     "access": "public"
   },

--- a/bindings/jupyter-modules/jupyter-ma-causal/package.json
+++ b/bindings/jupyter-modules/jupyter-ma-causal/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-ma-causal",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",

--- a/bindings/jupyter-modules/jupyter-ma-causal/package.json
+++ b/bindings/jupyter-modules/jupyter-ma-causal/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-ma-causal",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",

--- a/bindings/jupyter-modules/jupyter-ma-causal/package.json
+++ b/bindings/jupyter-modules/jupyter-ma-causal/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-ma-causal",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",

--- a/bindings/jupyter-modules/jupyter-ma-causal/package.json
+++ b/bindings/jupyter-modules/jupyter-ma-causal/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-ma-causal",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",

--- a/bindings/jupyter-modules/jupyter-ma-causal/package.json
+++ b/bindings/jupyter-modules/jupyter-ma-causal/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-ma-causal",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",

--- a/bindings/jupyter-modules/jupyter-ma-causal/package.json
+++ b/bindings/jupyter-modules/jupyter-ma-causal/package.json
@@ -7,6 +7,8 @@
     "url": "git+https://github.com/uber/manifold.git"
   },
   "main": "index.js",
+  "-------- Avoid publishing the prototype for now --------": "",
+  "private": true,
   "publishConfig": {
     "access": "public"
   },

--- a/bindings/jupyter-modules/jupyter-ma-causal/package.json
+++ b/bindings/jupyter-modules/jupyter-ma-causal/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-ma-causal",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",

--- a/bindings/jupyter-modules/jupyter-manifold/package.json
+++ b/bindings/jupyter-modules/jupyter-manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-manifold",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/manifold": "^1.1.4-alpha.0",
+    "@mlvis/manifold": "^1.1.4-alpha.1",
     "global": "^4.4.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",

--- a/bindings/jupyter-modules/jupyter-manifold/package.json
+++ b/bindings/jupyter-modules/jupyter-manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-manifold",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/manifold": "^1.1.4-alpha.2",
+    "@mlvis/manifold": "^1.1.4-alpha.3",
     "global": "^4.4.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",

--- a/bindings/jupyter-modules/jupyter-manifold/package.json
+++ b/bindings/jupyter-modules/jupyter-manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-manifold",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/manifold": "^1.1.4-alpha.1",
+    "@mlvis/manifold": "^1.1.4-alpha.2",
     "global": "^4.4.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",

--- a/bindings/jupyter-modules/jupyter-manifold/package.json
+++ b/bindings/jupyter-modules/jupyter-manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-manifold",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/manifold": "^1.1.4-alpha.5",
+    "@mlvis/manifold": "^1.1.4",
     "global": "^4.4.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",

--- a/bindings/jupyter-modules/jupyter-manifold/package.json
+++ b/bindings/jupyter-modules/jupyter-manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-manifold",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/manifold": "^1.1.3",
+    "@mlvis/manifold": "^1.1.4-alpha.0",
     "global": "^4.4.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",

--- a/bindings/jupyter-modules/jupyter-manifold/package.json
+++ b/bindings/jupyter-modules/jupyter-manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-manifold",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/manifold": "^1.1.4-alpha.3",
+    "@mlvis/manifold": "^1.1.4-alpha.4",
     "global": "^4.4.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",

--- a/bindings/jupyter-modules/jupyter-manifold/package.json
+++ b/bindings/jupyter-modules/jupyter-manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-manifold",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/manifold": "^1.1.4-alpha.4",
+    "@mlvis/manifold": "^1.1.4-alpha.5",
     "global": "^4.4.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",

--- a/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
+++ b/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-multi-way-plot",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.1",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.2",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "global": "^4.4.0",

--- a/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
+++ b/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-multi-way-plot",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/multi-way-plot": "^1.1.3",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.0",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "global": "^4.4.0",

--- a/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
+++ b/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-multi-way-plot",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.0",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.1",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "global": "^4.4.0",

--- a/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
+++ b/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-multi-way-plot",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.3",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.4",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "global": "^4.4.0",

--- a/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
+++ b/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-multi-way-plot",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.4",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.5",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "global": "^4.4.0",

--- a/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
+++ b/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-multi-way-plot",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.5",
+    "@mlvis/multi-way-plot": "^1.1.4",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "global": "^4.4.0",

--- a/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
+++ b/bindings/jupyter-modules/jupyter-multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/jupyter-multi-way-plot",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "transpile": "NODE_ENV=production babel src -d dist -s --config-file ../../../babel.config.js && yarn copy"
   },
   "dependencies": {
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.2",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.3",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "global": "^4.4.0",

--- a/lerna.json
+++ b/lerna.json
@@ -1,6 +1,6 @@
 {
   "lerna": "3.14.1",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "packages": [
     "modules/*",
     "bindings/jupyter-modules/*"

--- a/lerna.json
+++ b/lerna.json
@@ -1,6 +1,6 @@
 {
   "lerna": "3.14.1",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "packages": [
     "modules/*",
     "bindings/jupyter-modules/*"

--- a/lerna.json
+++ b/lerna.json
@@ -1,6 +1,6 @@
 {
   "lerna": "3.14.1",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "packages": [
     "modules/*",
     "bindings/jupyter-modules/*"

--- a/lerna.json
+++ b/lerna.json
@@ -1,6 +1,6 @@
 {
   "lerna": "3.14.1",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "packages": [
     "modules/*",
     "bindings/jupyter-modules/*"

--- a/lerna.json
+++ b/lerna.json
@@ -1,6 +1,6 @@
 {
   "lerna": "3.14.1",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "packages": [
     "modules/*",
     "bindings/jupyter-modules/*"

--- a/lerna.json
+++ b/lerna.json
@@ -1,6 +1,6 @@
 {
   "lerna": "3.14.1",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "packages": [
     "modules/*",
     "bindings/jupyter-modules/*"

--- a/lerna.json
+++ b/lerna.json
@@ -1,6 +1,6 @@
 {
   "lerna": "3.14.1",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "packages": [
     "modules/*",
     "bindings/jupyter-modules/*"

--- a/modules/feature-list-view/package.json
+++ b/modules/feature-list-view/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/feature-list-view",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "license": "Apache-2.0",
   "description": "Feature List View component of the Manifold",
   "publishConfig": {
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.3",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.4",
     "d3-scale": "^3.0.0",
     "numeral": "^2.0.6",
     "prop-types": "^15.7.2",

--- a/modules/feature-list-view/package.json
+++ b/modules/feature-list-view/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/feature-list-view",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "license": "Apache-2.0",
   "description": "Feature List View component of the Manifold",
   "publishConfig": {
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.3",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.0",
     "d3-scale": "^3.0.0",
     "numeral": "^2.0.6",
     "prop-types": "^15.7.2",

--- a/modules/feature-list-view/package.json
+++ b/modules/feature-list-view/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/feature-list-view",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "license": "Apache-2.0",
   "description": "Feature List View component of the Manifold",
   "publishConfig": {
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.4",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.5",
     "d3-scale": "^3.0.0",
     "numeral": "^2.0.6",
     "prop-types": "^15.7.2",

--- a/modules/feature-list-view/package.json
+++ b/modules/feature-list-view/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/feature-list-view",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "license": "Apache-2.0",
   "description": "Feature List View component of the Manifold",
   "publishConfig": {
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.2",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.3",
     "d3-scale": "^3.0.0",
     "numeral": "^2.0.6",
     "prop-types": "^15.7.2",

--- a/modules/feature-list-view/package.json
+++ b/modules/feature-list-view/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/feature-list-view",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "license": "Apache-2.0",
   "description": "Feature List View component of the Manifold",
   "publishConfig": {
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.5",
+    "@mlvis/mlvis-common": "^1.1.4",
     "d3-scale": "^3.0.0",
     "numeral": "^2.0.6",
     "prop-types": "^15.7.2",

--- a/modules/feature-list-view/package.json
+++ b/modules/feature-list-view/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/feature-list-view",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "license": "Apache-2.0",
   "description": "Feature List View component of the Manifold",
   "publishConfig": {
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.1",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.2",
     "d3-scale": "^3.0.0",
     "numeral": "^2.0.6",
     "prop-types": "^15.7.2",

--- a/modules/feature-list-view/package.json
+++ b/modules/feature-list-view/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/feature-list-view",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "license": "Apache-2.0",
   "description": "Feature List View component of the Manifold",
   "publishConfig": {
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.0",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.1",
     "d3-scale": "^3.0.0",
     "numeral": "^2.0.6",
     "prop-types": "^15.7.2",

--- a/modules/graph-builder/package.json
+++ b/modules/graph-builder/package.json
@@ -6,7 +6,7 @@
     "type": "git",
     "url": "git+https://github.com/uber/manifold.git"
   },
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "author": "Javid Hsueh",
   "main": "index.js",
   "publishConfig": {

--- a/modules/graph-builder/package.json
+++ b/modules/graph-builder/package.json
@@ -6,7 +6,7 @@
     "type": "git",
     "url": "git+https://github.com/uber/manifold.git"
   },
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "author": "Javid Hsueh",
   "main": "index.js",
   "publishConfig": {

--- a/modules/graph-builder/package.json
+++ b/modules/graph-builder/package.json
@@ -6,7 +6,7 @@
     "type": "git",
     "url": "git+https://github.com/uber/manifold.git"
   },
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "author": "Javid Hsueh",
   "main": "index.js",
   "publishConfig": {

--- a/modules/graph-builder/package.json
+++ b/modules/graph-builder/package.json
@@ -6,7 +6,7 @@
     "type": "git",
     "url": "git+https://github.com/uber/manifold.git"
   },
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "author": "Javid Hsueh",
   "main": "index.js",
   "publishConfig": {

--- a/modules/graph-builder/package.json
+++ b/modules/graph-builder/package.json
@@ -6,7 +6,7 @@
     "type": "git",
     "url": "git+https://github.com/uber/manifold.git"
   },
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "author": "Javid Hsueh",
   "main": "index.js",
   "publishConfig": {

--- a/modules/graph-builder/package.json
+++ b/modules/graph-builder/package.json
@@ -6,7 +6,7 @@
     "type": "git",
     "url": "git+https://github.com/uber/manifold.git"
   },
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "author": "Javid Hsueh",
   "main": "index.js",
   "publishConfig": {

--- a/modules/graph-builder/package.json
+++ b/modules/graph-builder/package.json
@@ -6,7 +6,7 @@
     "type": "git",
     "url": "git+https://github.com/uber/manifold.git"
   },
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "author": "Javid Hsueh",
   "main": "index.js",
   "publishConfig": {

--- a/modules/manifold/package.json
+++ b/modules/manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/manifold",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "publishConfig": {
     "access": "public"
   },
@@ -24,9 +24,9 @@
     "@babel/polyfill": "^7.0.0",
     "@loaders.gl/arrow": "^1.2.0-alpha.1",
     "@loaders.gl/core": "^1.1.0",
-    "@mlvis/feature-list-view": "^1.1.4-alpha.3",
-    "@mlvis/mlvis-common": "^1.1.4-alpha.3",
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.3",
+    "@mlvis/feature-list-view": "^1.1.4-alpha.4",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.4",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.4",
     "@tensorflow/tfjs": "^0.15.3",
     "baseui": "^9.2.0",
     "d3-color": "^1.3.0",

--- a/modules/manifold/package.json
+++ b/modules/manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/manifold",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "publishConfig": {
     "access": "public"
   },
@@ -24,9 +24,9 @@
     "@babel/polyfill": "^7.0.0",
     "@loaders.gl/arrow": "^1.2.0-alpha.1",
     "@loaders.gl/core": "^1.1.0",
-    "@mlvis/feature-list-view": "^1.1.3",
-    "@mlvis/mlvis-common": "^1.1.3",
-    "@mlvis/multi-way-plot": "^1.1.3",
+    "@mlvis/feature-list-view": "^1.1.4-alpha.0",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.0",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.0",
     "@tensorflow/tfjs": "^0.15.3",
     "baseui": "^9.2.0",
     "d3-color": "^1.3.0",

--- a/modules/manifold/package.json
+++ b/modules/manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/manifold",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "publishConfig": {
     "access": "public"
   },
@@ -24,9 +24,9 @@
     "@babel/polyfill": "^7.0.0",
     "@loaders.gl/arrow": "^1.2.0-alpha.1",
     "@loaders.gl/core": "^1.1.0",
-    "@mlvis/feature-list-view": "^1.1.4-alpha.4",
-    "@mlvis/mlvis-common": "^1.1.4-alpha.4",
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.4",
+    "@mlvis/feature-list-view": "^1.1.4-alpha.5",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.5",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.5",
     "@tensorflow/tfjs": "^0.15.3",
     "baseui": "^9.2.0",
     "d3-color": "^1.3.0",

--- a/modules/manifold/package.json
+++ b/modules/manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/manifold",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "publishConfig": {
     "access": "public"
   },
@@ -24,9 +24,9 @@
     "@babel/polyfill": "^7.0.0",
     "@loaders.gl/arrow": "^1.2.0-alpha.1",
     "@loaders.gl/core": "^1.1.0",
-    "@mlvis/feature-list-view": "^1.1.4-alpha.5",
-    "@mlvis/mlvis-common": "^1.1.4-alpha.5",
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.5",
+    "@mlvis/feature-list-view": "^1.1.4",
+    "@mlvis/mlvis-common": "^1.1.4",
+    "@mlvis/multi-way-plot": "^1.1.4",
     "@tensorflow/tfjs": "^0.15.3",
     "baseui": "^9.2.0",
     "d3-color": "^1.3.0",

--- a/modules/manifold/package.json
+++ b/modules/manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/manifold",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "publishConfig": {
     "access": "public"
   },
@@ -24,9 +24,9 @@
     "@babel/polyfill": "^7.0.0",
     "@loaders.gl/arrow": "^1.2.0-alpha.1",
     "@loaders.gl/core": "^1.1.0",
-    "@mlvis/feature-list-view": "^1.1.4-alpha.1",
-    "@mlvis/mlvis-common": "^1.1.4-alpha.1",
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.1",
+    "@mlvis/feature-list-view": "^1.1.4-alpha.2",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.2",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.2",
     "@tensorflow/tfjs": "^0.15.3",
     "baseui": "^9.2.0",
     "d3-color": "^1.3.0",

--- a/modules/manifold/package.json
+++ b/modules/manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/manifold",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "publishConfig": {
     "access": "public"
   },
@@ -24,9 +24,9 @@
     "@babel/polyfill": "^7.0.0",
     "@loaders.gl/arrow": "^1.2.0-alpha.1",
     "@loaders.gl/core": "^1.1.0",
-    "@mlvis/feature-list-view": "^1.1.4-alpha.2",
-    "@mlvis/mlvis-common": "^1.1.4-alpha.2",
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.2",
+    "@mlvis/feature-list-view": "^1.1.4-alpha.3",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.3",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.3",
     "@tensorflow/tfjs": "^0.15.3",
     "baseui": "^9.2.0",
     "d3-color": "^1.3.0",

--- a/modules/manifold/package.json
+++ b/modules/manifold/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/manifold",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "publishConfig": {
     "access": "public"
   },
@@ -24,9 +24,9 @@
     "@babel/polyfill": "^7.0.0",
     "@loaders.gl/arrow": "^1.2.0-alpha.1",
     "@loaders.gl/core": "^1.1.0",
-    "@mlvis/feature-list-view": "^1.1.4-alpha.0",
-    "@mlvis/mlvis-common": "^1.1.4-alpha.0",
-    "@mlvis/multi-way-plot": "^1.1.4-alpha.0",
+    "@mlvis/feature-list-view": "^1.1.4-alpha.1",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.1",
+    "@mlvis/multi-way-plot": "^1.1.4-alpha.1",
     "@tensorflow/tfjs": "^0.15.3",
     "baseui": "^9.2.0",
     "d3-color": "^1.3.0",

--- a/modules/mlvis-common/package.json
+++ b/modules/mlvis-common/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/mlvis-common",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "license": "Apache-2.0",
   "publishConfig": {
     "access": "public"

--- a/modules/mlvis-common/package.json
+++ b/modules/mlvis-common/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/mlvis-common",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "license": "Apache-2.0",
   "publishConfig": {
     "access": "public"

--- a/modules/mlvis-common/package.json
+++ b/modules/mlvis-common/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/mlvis-common",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "license": "Apache-2.0",
   "publishConfig": {
     "access": "public"

--- a/modules/mlvis-common/package.json
+++ b/modules/mlvis-common/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/mlvis-common",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "license": "Apache-2.0",
   "publishConfig": {
     "access": "public"

--- a/modules/mlvis-common/package.json
+++ b/modules/mlvis-common/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/mlvis-common",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "license": "Apache-2.0",
   "publishConfig": {
     "access": "public"

--- a/modules/mlvis-common/package.json
+++ b/modules/mlvis-common/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/mlvis-common",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "license": "Apache-2.0",
   "publishConfig": {
     "access": "public"

--- a/modules/mlvis-common/package.json
+++ b/modules/mlvis-common/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/mlvis-common",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "license": "Apache-2.0",
   "publishConfig": {
     "access": "public"

--- a/modules/multi-way-plot/package.json
+++ b/modules/multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/multi-way-plot",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "publishConfig": {
     "access": "public"
   },
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.5",
+    "@mlvis/mlvis-common": "^1.1.4",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "d3-shape": "^1.3.5",

--- a/modules/multi-way-plot/package.json
+++ b/modules/multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/multi-way-plot",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "publishConfig": {
     "access": "public"
   },
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.2",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.3",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "d3-shape": "^1.3.5",

--- a/modules/multi-way-plot/package.json
+++ b/modules/multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/multi-way-plot",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "publishConfig": {
     "access": "public"
   },
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.3",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.0",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "d3-shape": "^1.3.5",

--- a/modules/multi-way-plot/package.json
+++ b/modules/multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/multi-way-plot",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "publishConfig": {
     "access": "public"
   },
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.4",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.5",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "d3-shape": "^1.3.5",

--- a/modules/multi-way-plot/package.json
+++ b/modules/multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/multi-way-plot",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "publishConfig": {
     "access": "public"
   },
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.0",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.1",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "d3-shape": "^1.3.5",

--- a/modules/multi-way-plot/package.json
+++ b/modules/multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/multi-way-plot",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "publishConfig": {
     "access": "public"
   },
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.1",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.2",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "d3-shape": "^1.3.5",

--- a/modules/multi-way-plot/package.json
+++ b/modules/multi-way-plot/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mlvis/multi-way-plot",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "publishConfig": {
     "access": "public"
   },
@@ -23,7 +23,7 @@
     "es6"
   ],
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.3",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.4",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3",
     "d3-shape": "^1.3.5",

--- a/modules/stacked-calendar/package.json
+++ b/modules/stacked-calendar/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@mlvis/stacked-calendar",
   "description": "A day-hour calendar view with stacked charts for each hour",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -10,7 +10,7 @@
   "author": "Yang Wang",
   "main": "index.js",
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.0",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.1",
     "d3-array": "^2.0.3",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3"

--- a/modules/stacked-calendar/package.json
+++ b/modules/stacked-calendar/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@mlvis/stacked-calendar",
   "description": "A day-hour calendar view with stacked charts for each hour",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -10,7 +10,7 @@
   "author": "Yang Wang",
   "main": "index.js",
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.3",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.4",
     "d3-array": "^2.0.3",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3"

--- a/modules/stacked-calendar/package.json
+++ b/modules/stacked-calendar/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@mlvis/stacked-calendar",
   "description": "A day-hour calendar view with stacked charts for each hour",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -10,7 +10,7 @@
   "author": "Yang Wang",
   "main": "index.js",
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.1",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.2",
     "d3-array": "^2.0.3",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3"

--- a/modules/stacked-calendar/package.json
+++ b/modules/stacked-calendar/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@mlvis/stacked-calendar",
   "description": "A day-hour calendar view with stacked charts for each hour",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -10,7 +10,7 @@
   "author": "Yang Wang",
   "main": "index.js",
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.5",
+    "@mlvis/mlvis-common": "^1.1.4",
     "d3-array": "^2.0.3",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3"

--- a/modules/stacked-calendar/package.json
+++ b/modules/stacked-calendar/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@mlvis/stacked-calendar",
   "description": "A day-hour calendar view with stacked charts for each hour",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -10,7 +10,7 @@
   "author": "Yang Wang",
   "main": "index.js",
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.2",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.3",
     "d3-array": "^2.0.3",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3"

--- a/modules/stacked-calendar/package.json
+++ b/modules/stacked-calendar/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@mlvis/stacked-calendar",
   "description": "A day-hour calendar view with stacked charts for each hour",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -10,7 +10,7 @@
   "author": "Yang Wang",
   "main": "index.js",
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.4-alpha.4",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.5",
     "d3-array": "^2.0.3",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3"

--- a/modules/stacked-calendar/package.json
+++ b/modules/stacked-calendar/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@mlvis/stacked-calendar",
   "description": "A day-hour calendar view with stacked charts for each hour",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
@@ -10,7 +10,7 @@
   "author": "Yang Wang",
   "main": "index.js",
   "dependencies": {
-    "@mlvis/mlvis-common": "^1.1.3",
+    "@mlvis/mlvis-common": "^1.1.4-alpha.0",
     "d3-array": "^2.0.3",
     "d3-scale": "^3.0.0",
     "d3-scale-chromatic": "^1.3.3"

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "manifold",
-  "version": "1.1.4-alpha.5",
+  "version": "1.1.4",
   "description": "A suite of machine learning visualization utilities",
   "license": "Apache-2.0",
   "private": true,

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "manifold",
-  "version": "1.1.4-alpha.1",
+  "version": "1.1.4-alpha.2",
   "description": "A suite of machine learning visualization utilities",
   "license": "Apache-2.0",
   "private": true,

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "manifold",
-  "version": "1.1.3",
+  "version": "1.1.4-alpha.0",
   "description": "A suite of machine learning visualization utilities",
   "license": "Apache-2.0",
   "private": true,

--- a/package.json
+++ b/package.json
@@ -84,7 +84,7 @@
     "lerna:bump": "lerna version --no-push && npm run lerna:sync && npm run lerna:git",
     "lerna:sync": "jq -M --argfile lerna lerna.json '.version = $lerna.version' package.json > tmp.json && mv tmp.json package.json",
     "lerna:git": "git add package.json && git commit --amend",
-    "lerna:publish": "lerna exec cp package.json dist && lerna exec npm publish dist",
+    "lerna:publish": "lerna exec cp package.json dist && lerna exec --no-private npm publish dist",
     "-------- WEBSITE --------": "",
     "install:all": "yarn clean && yarn install && yarn install:web",
     "install:web": "yarn install:example && cd website && yarn install",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "manifold",
-  "version": "1.1.4-alpha.0",
+  "version": "1.1.4-alpha.1",
   "description": "A suite of machine learning visualization utilities",
   "license": "Apache-2.0",
   "private": true,

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "manifold",
-  "version": "1.1.4-alpha.4",
+  "version": "1.1.4-alpha.5",
   "description": "A suite of machine learning visualization utilities",
   "license": "Apache-2.0",
   "private": true,

--- a/package.json
+++ b/package.json
@@ -84,7 +84,7 @@
     "lerna:bump": "lerna version --no-push && npm run lerna:sync && npm run lerna:git",
     "lerna:sync": "jq -M --argfile lerna lerna.json '.version = $lerna.version' package.json > tmp.json && mv tmp.json package.json",
     "lerna:git": "git add package.json && git commit --amend",
-    "lerna:publish": "lerna exec cp package.json dist && lerna exec --no-private npm publish dist",
+    "lerna:publish": "lerna exec cp package.json dist && lerna exec npm publish dist",
     "-------- WEBSITE --------": "",
     "install:all": "yarn clean && yarn install && yarn install:web",
     "install:web": "yarn install:example && cd website && yarn install",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "manifold",
-  "version": "1.1.4-alpha.2",
+  "version": "1.1.4-alpha.3",
   "description": "A suite of machine learning visualization utilities",
   "license": "Apache-2.0",
   "private": true,

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "manifold",
-  "version": "1.1.4-alpha.3",
+  "version": "1.1.4-alpha.4",
   "description": "A suite of machine learning visualization utilities",
   "license": "Apache-2.0",
   "private": true,

